### PR TITLE
tests: net: socket: websocket move to new ztest API

### PR DESCRIPTION
**Pull request by hunterhu3000**
Move net/socket/websocket tests to use new ztest API

Signed-off-by: Hu Zhenyu <zhenyu.hu@intel.com>

### CODE REVIEW

**Review by rlubos — DISMISSED** [review]

**Review by enjiamai — DISMISSED** [review]

**Review by nashif — CHANGES_REQUESTED** [review]

File: tests/net/socket/websocket/src/main.c [hunk 1]
```diff
@@ -175,52 +175,52 @@ static void test_recv(int count)
 	zassert_equal(remaining, 0, "Msg not empty");
 }
 
-static void test_recv_1_byte(void)
+ZTEST(websocket_test_suite, test_recv_1_byte)
```
[the remark this answers is not in the document]
Comment (hunterhu3000) [reply tests/net/socket/websocket/src/main.c]: Updated. Thanks to review.

**Comment by hunterhu3000** [comment]
CONFIG_ZTEST_SHUFFLE=y can be turned on

**Review by rlubos — DISMISSED** [review]

**Review by enjiamai — DISMISSED** [review]

**Review by enjiamai — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Comment by hunterhu3000** [comment]
@nashif Thanks to review and +1, so that the PR can be merged into the mainline.

**Comment by hunterhu3000** [comment]
@enjiamai  the numbers in the function name is not added by me, it was there before.
@nashif Thank to review and +1, so that the patch can be merged to mainline.

**Review by stephanosio — APPROVED** [review]

**Comment by hunterhu3000** [comment]
@nashif can we merge this PR?

**Pull request closed without merging**